lib/atob.js

```diff
@@ -39,7 +39,15 @@ function polyfill(input) {
     return output;
 }
 
-export default (typeof window !== "undefined" &&
-    window.atob &&
-    window.atob.bind(window)) ||
-polyfill;
\ No newline at end of file
+// globalThis supported and atob implemented
+if (typeof globalThis !== "undefined" && globalThis.atob) {
+    export globalThis.atob.bind(globalThis);
+} 
+// no globalThis support, however window present and atob implemented
+else if (typeof window !== "undefined" && window.atob) {
+    export window.atob.bind(window);
+} 
```

Comment:
Perhaps we could remove the whole check on `window`, and just assume `globalThis` only?